dot/peerset/peerset_test.go

```diff
@@ -11,57 +11,103 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestPeerSetBanned(t *testing.T) {
+const testSetID = 0
```

Comment:
nit inline it in each test where it's needed to reduce the Go package API pollution (even if it's just the test API one)